SimPEG/EM/Static/Utils/StaticUtils.py

```diff
@@ -184,16 +184,12 @@ def gen_DCIPsurvey(endl, mesh, surveyType, a, b, n):
         :param endl -> input endpoints [x1, y1, z1, x2, y2, z2]
         :object mesh -> SimPEG mesh object
         :switch surveyType -> "dipole-dipole" (dipole-dipole) | "pole-dipole" (pole-dipole) | 'gradient'
-        : param a, n -> pole seperation, number of rx dipoles per tx
+        :param a, n -> pole seperation, number of rx dipoles per tx
```

Comment:
I realize that you didn't author this, but @thast, would you be willing to update this docstring so it follows standard sphinx? 
```
:param int a: pole seperation 
```